packages/react-aria-components/src/GridList.tsx

```diff
@@ -257,8 +257,9 @@ function GridListRow({item}) {
   );
 
   let {hoverProps, isHovered} = useHover({
-    isDisabled: !states.allowsSelection && !states.hasAction
+    isDisabled: state.disabledKeys.has(item.key)
```

Comment:
hmm I think we need to use `states.isDisabled` perhaps? That takes into account `disabledBehavior` as well. An item might be in `disabledKeys` but still have an action, therefore it's interactive.

Reply:
hrm, I guess we'd also need to account for if there is a href as well?